assets/js/googlesitekit/modules/datastore/modules.test.js

```diff
@@ -182,7 +193,7 @@ describe( 'core/modules modules', () => {
 
 				// The third request to update the modules shouldn't be called, because the
 				// activation request failed.
-				expect( fetchMock ).toHaveFetchedTimes( 2 );
+				expect( fetchMock ).toHaveBeenCalledTimes( 3 );
```

Comment:
Is this correct? The comment makes it sound like it should maybe have only fetched 2 times?